app/controllers/hooks_controller.rb

```diff
@@ -21,6 +21,7 @@ def send_github_payload_to_job(payload_body)
     github_event = request.env["HTTP_X_GITHUB_EVENT"]
     return unless GitHub::WebHook::ACCEPTED_EVENTS.include?(github_event)
     "#{github_event}_event_job".classify.constantize.perform_later(payload_body)
+    update_last_webhook_recieved(payload_body)
```

Comment:
Maybe we should invoke this every time we revive a webhook from GitHub instead of if the webhook event type is one of the `GitHub::WebHook::ACCEPTED_EVENTS`?